redhat-upgrade-tool.py

```diff
@@ -326,7 +258,7 @@ def handle_sigwinch(signum, frame):
     # Save the repo configuration
     f.save_repo_configs()
 
-    #dump all configuration to upgrade.conf, other tools need to know
+    # Dump all configuration to upgrade.conf, other tools need to know
```

Comment:
I do not understand

Reply:
This change is really just formal and was not necessary. But I did it:
- to follow PEP formating (https://www.python.org/dev/peps/pep-0008/#block-comments)
- to have the comment consistent with the other comments (have a space after # and start with capital letter)